subjects/ai/kaggle-titanic/README.md

```diff
@@ -46,10 +46,8 @@ project
 |   |   gender_submission.csv
 │
 └───notebook
-│   │   EDA.ipynb
-|
-|───scripts
-│
+│   │   main.ipynb
```

Comment:
I think this is an extra  `|`

Reply:
It defines the scope of the file inside the scripts folder.